talisman.go

```diff
@@ -58,6 +58,13 @@ func main() {
 		os.Exit(0)
 	}
 
+	if flag.NFlag() == 0 {
+		fmt.Println(
+			"Usage \"talisman [option]\"\nOptions:\n\t\"-scan\" -> scan git history\n\t\"-v\" -> check Talisman version",
+		)
```

Comment:
Please consider using flag.PrintDefaults, at least.
Stretch goals: switch out flag with one of spf13's packages [pflag](https://github.com/spf13/pflag), [cobra](https://github.com/spf13/cobra) or [viper](https://github.com/spf13/viper)